test/unit/region.spec.js

```diff
@@ -901,18 +901,15 @@ describe('region', function() {
         })
       });
 
-      this.MyView = Backbone.View.extend({
-        render: function() {
-          $(this.el).html('some content');
-        },
-
-        destroy: function() {}
+      this.MyView = Backbone.Marionette.View.extend({
```

Comment:
these tests were/are awkward. 
I think the point of them is to check that region doesn't double destroy a view.  For Marionette.View this is impossible since `destroy` guards against this, but for a `Backbone.View` it is possible to double remove.  I don't know that changing this to a `Marionette.View` is helpful